docs/source/examples/import_classes.py

```diff
@@ -0,0 +1,60 @@
+from gpkit import Variable, Model
```

Comment:
i just use it to support two examples which both draw on these same three classes. will add a docstring and edit the name